### PR TITLE
COD-28 A simple UI for webapp.

### PR DESCRIPTION
**Pull request by zohaib194**

### CODE REVIEW

File: frontend/js/GUI/Window.js [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+
+    ImGui.StyleColorsDark();
+    //ImGui.StyleColorsClassic();
+
+
+    let done = false;
+    window.requestAnimationFrame(ImGuiLoop);
+
+    /**
+     * Function for making menubar object.
```
Comment (eldarht) [inline frontend/js/GUI/Window.js]: Could we change "object" to window or something that shows it being a imgui menuebar

Comment (eldarht) [reply frontend/js/GUI/Window.js]: Same with other windows.

File: frontend/js/GUI/Window.js [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+     *
+     * @return     {Object}  object contains menubar functions.
+     */
+    let menubar = (function (){
```
[the remark this answers is not in the document]
Comment (zohaib194) [reply frontend/js/GUI/Window.js]: agree.

File: frontend/html/3DView.html [hunk 3]
```diff
@@ -12,13 +12,24 @@
...
         <!-- THREE.js + (soon to come) ImGUI imports -->
         <script src="../js/treejs/three.js"></script>
         <script src="../js/treejs/controls/OrbitControls.js"></script>
+
+        <script type="text/javascript" src="../js/GUI/Menubar.js"></script>
+        <script type="text/javascript" src="../js/GUI/FiletreeWindow.js"></script>
+        <script type="text/javascript" src="../js/GUI/CodeInspectionWindow.js"></script>
+        <script type="text/javascript" src="../js/GUI/NamespaceWindow.js"></script>
+        <script type="text/javascript" src="../js/GUI/QualityMetricsWindow.js"></script>
+        <script type="text/javascript" src="../js/GUI/WindowManager.js"></script>
```
Comment (Xillez) [inline frontend/html/3DView.html]: Dont need the type specifier. Adds unnecessary information

File: README.md [hunk 4]
```diff
@@ -10,6 +10,14 @@ CodebaseVisulizer3D is a bachelor project with the goal of making it easier to g
 - Eldar Hauge Torkelsen - 473180
 
 # Install guide
+- Clone the repository.
+
+### Setup frontend
+- Download and Install [Emscripten](http://kripken.github.io/emscripten-site/docs/getting_started/downloads.html)
+- Navigate to the root folder of repository and run the following command.
+    - git submodule update --init --recursive
```
Comment (eldarht) [inline README.md]: Use command block for commands.

**Pull request merged**